rules/cross-platform/guided_onborading_sample_rule.toml

```diff
@@ -0,0 +1,60 @@
+[metadata]
+creation_date = "2022/09/22"
+maturity = "production"
+updated_date = "2022/09/22"
+min_stack_comments = "Guided Onboarding will be available in Elastic 8.6+"
+min_stack_version = "8.6.0"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule helps you test and practice using alerts with Elastic Security as you get set up. It’s not a sign of threat
+activity.
+"""
+false_positives = [
+    """
+    This rule is not looking for threat activity. Disable the rule if you're already familiar with alerts.
+    """,
+]
+max_signals = 5
+interval = "24h"
+enabled = false
+from = "now-24h"
+index = [
+    "apm-*-transaction*",
+    "auditbeat-*",
+    "endgame-*",
+    "filebeat-*",
+    "logs-*",
+    "packetbeat-*",
+    "traces-apm*",
+    "winlogbeat-*",
+    "-*elastic-cloud-logs-*"
```

Comment:
Is this right? Leading with a `-`?

Reply:
It's how it shows in the Security Solution when you create a new rule.

Reply:
Leading "-" means that index is excluded. Done on purpose be default at rule creation.